src/connectors/README.md

```diff
@@ -13,8 +13,8 @@ Connects provided component with a wrapper behind internal PWA routing actions.
 Everywhere as a React component rendered within the PWA app.
 
 #### Props provided
-- `historyPush(pathname: string)` - Opens new page in a safe way. Detects link types. Internal links (deep links) which are handled by the PWA app. External links are handled by opening the in-app-browser.
-- `historyReplace(pathname:string)` - Replaces current page with provided pathname if pathname is an internal link. If pathname is external link it works exactly as `historyPush`. If you intend to replace your current page with in-app-browser, use combination of `historyPush` and `historyPop` instead.
+- `historyPush(pathname: string, state: object[optional], silent: boolean[optional])` - Opens new page in a safe way. Detects link types. Internal links (deep links) which are handled by the PWA app. External links are handled by opening the in-app-browser.
```

Comment:
You've changed the functions signatures, but forgot about the documentation. Please always make self-reviews.

Reply:
Updated this readme file